tests/acceptance/NewsletterCreationCest.php

```diff
@@ -60,27 +55,23 @@ function createStandardNewsletter(\AcceptanceTester $I) {
     $I->click('[data-automation-id="new_email"]');
 
     // step 1 - select notification type
-    $I->seeInCurrentUrl('#/new');
     $I->click('[data-automation-id="create_standard"]');
 
     // step 2 - select template
     $standard_template = '[data-automation-id="select_template_0"]';
     $I->waitForElement($standard_template);
     $I->see('Newsletters', ['css' => 'a.current']);
-    $I->seeInCurrentUrl('#/template');
     $I->click($standard_template);
 
     // step 3 - design newsletter (update subject)
     $title_element = '[data-automation-id="newsletter_title"]';
     $I->waitForElement($title_element);
-    $I->seeInCurrentUrl('mailpoet-newsletter-editor');
     $I->fillField($title_element, $newsletter_title);
     $I->click('Next');
 
     // step 4 - Choose list and send
     $send_form_element = '[data-automation-id="newsletter_send_form"]';
     $I->waitForElement($send_form_element);
-    $I->seeInCurrentUrl('mailpoet-newsletters#/send/');
     $I->selectOptionInSelect2('WordPress Users');
     $I->click('Send');
```

Comment:
Not for this Pull request, but we don't assert at the end of this test. We should probably check that there was no error and the newsletter was sent.